doc.go

```diff
@@ -435,6 +436,11 @@ A custom 'delete' command can be defined to override this default.
 Rename the current file using the builtin method.
 A custom 'rename' command can be defined to override this default.
 
+	mkdir          (modal)   (default 'n')
```

Comment:
This should be changed to `a` as well:

```suggestion
	mkdir          (modal)   (default 'a')
```

And remember to run `go generate` again so the changes are reflected.

Reply:
Oops! Completely forgot about that line haha